ospo-mindmap/Content/ospomindmap.md

```diff
@@ -356,3 +356,25 @@ incubation and post launch
 
 - Help an organization understand the open source technical stack, compliance and opportunities of the target company and related risk as part of the due diligence process
 - Ensure contractors (outsourced developement) follow the organization's open source policies and processes
+
+## 🤝 Stakeholders
+
+- Legal
+  - License compliance
+  - Policies and guidelines
+  - Mergers and acquisitions
+  - Investment due diligence
+  - Contribution reviews
+  - Trademark searches for new open source projects
+- Security
+  - Software supply chain risk management
+- Engineering & Product
+  - Adopt tooling
+  - Follow policy and guidelines
```

Comment:
One addition: An OSPO is often involved in creating, advertising or providing open source specific training. This could be categorized under Engineering or even a dedicated "Learning and Development" department as additional stakeholder.

```suggestion
  - Follow policy and guidelines
Learning and Development
  - Training on open source ways-of-working
```